### PR TITLE
Renamed resetPassword to asResetPassword

### PR DESCRIPTION
**Pull request by vromash**

### CODE REVIEW

File: src/etc/schema.graphqls [hunk 1]
```diff
@@ -16,7 +16,7 @@ type Mutation {
     generateCustomerToken(email: String!, password: String!, guest_quote_id: String): CustomerToken @resolver(class: "\\ScandiPWA\\CustomerGraphQl\\Model\\Resolver\\GenerateCustomerToken") @doc(description:"Retrieve the customer token")
     resendConfirmationEmail(email: String!): CustomerActionConfirmationType @resolver(class: "\\ScandiPWA\\CustomerGraphQl\\Model\\Resolver\\ResendConfirmationEmail") @doc(description:"Resend customer confirmation")
     forgotPassword(email: String!): CustomerActionConfirmationType @resolver(class: "\\ScandiPWA\\CustomerGraphQl\\Model\\Resolver\\ForgotPassword") @doc(description:"Resend customer confirmation")
-    resetPassword(password: String!, token: String!, password_confirmation: String!): ResetPasswordType @resolver(class: "\\ScandiPWA\\CustomerGraphQl\\Model\\Resolver\\ResetPassword") @doc(description:"Resend customer confirmation")
+    asResetPassword(password: String!, token: String!, password_confirmation: String!): ResetPasswordType @resolver(class: "\\ScandiPWA\\CustomerGraphQl\\Model\\Resolver\\ResetPassword") @doc(description:"Resend customer confirmation")
```
Comment (alfredsgenkins) [inline src/etc/schema.graphqls:19]: The `as` seems like a bad prefix to me. Can we do the `sw` prefix, like we did for some other endpoints we matched in Magento 2?

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]
Where is the related front-end PR?

File: src/etc/schema.graphqls (hunk 1 above)
Comment (alfredsgenkins) [reply src/etc/schema.graphqls:19]: Check the `quote-graphql` for example.

**Comment by alfredsgenkins** [comment]
Please change this PR base - the `master` must be used.

**Comment by vromash** [comment]
Here is PR for frontend: https://github.com/scandipwa/scandipwa/pull/2074
`master` branch seems to be outdated and `2.x` is default.

**Pull request merged**